com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/VertexProgramBuilder.java

```diff
@@ -22,17 +22,16 @@
 import com.dynamo.bob.Platform;
 import com.dynamo.bob.Project;
 import com.dynamo.bob.fs.DefaultFileSystem;
-import com.dynamo.bob.pipeline.ShaderUtil.ES2ToES3Converter;
 import com.dynamo.bob.pipeline.IShaderCompiler;
 import com.dynamo.bob.pipeline.ShaderPreprocessor;
 
+import com.dynamo.graphics.proto.Graphics.ShaderDesc;
+
 import org.apache.commons.cli.CommandLine;
 
 @BuilderParams(name = "VertexProgram", inExts = ".vp", outExt = ".vpc")
 public class VertexProgramBuilder extends ShaderProgramBuilder {
```

Comment:
I don't think we _need_ these specific builds any more, all shaders work exactly the same. The difference is the type, which we can derive from the extension..